easybuild/easyblocks/o/openfoam.py

```diff
@@ -281,6 +281,9 @@ def build_step(self):
                 "Cleaning .*",
             ]
             run_cmd_qa(cmd_tmpl % 'Allwmake.firstInstall', qa, no_qa=noqa, log_all=True, simple=True)
+        elif LooseVersion(self.version) >= LooseVersion('1600'):
```

Comment:
Then you should modify the version check to `>= LooseVersion('1606')`?

See code snippet above for a suggestion on how to avoid copy-pasting.